files/ko/web/javascript/reference/global_objects/intl/supportedvaluesof/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Intl.supportedValuesOf()
+slug: Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf
+tags:
+  - Internationalization
+  - Intl
+  - JavaScript
+  - Method
+  - Reference
+  - supportedValuesOf
+browser-compat: javascript.builtins.Intl.supportedValuesOf
```

Comment:
Hi @bangina, we can remove this part, see #7412

```suggestion
```

Reply:
@yin1999 Thank you!